src/gcinfo/CMakeLists.txt

```diff
@@ -23,5 +23,15 @@ endif(CLR_CMAKE_PLATFORM_UNIX)
 add_subdirectory(lib)
 add_subdirectory(crossgen)
 
+if (CLR_CMAKE_PLATFORM_ARCH_I386)
+    # On x86, build RyuJIT/ARM32 cross-compiling altjit.
+    add_subdirectory(arm_altjit)
```

Comment:
I don't love that we have `altjit` in the paths here, but I don't have a great alternative. `arm{,64}_cross_gcinfo`, maybe?